src/App.vue

```diff
@@ -407,16 +409,18 @@ export default {
     },
 
     /** Fetches and stores the incorp app data. */
```

Comment:
Might as well update this too :)

Reply:
I know naming is hard... Is this really a draft app?

Reply:
haha can do!
Abbreviated for draft application, i'm assuming that's what `app` was abbreviated for here previously.
Should we change?

Reply:
No, it was "incorp app" before (IA).

Reply:
I'm warming up to the idea that it's either an "incorporation application" or a "registration application". (Is this correct?)

Since I can't think of a better name atm, maybe just leave it?

Reply:
I'm with ya, for the lack of a better term might just leave it. 
Incorporations and Registrations are both 'applications' of sorts 🤷 
I'm not sure of a term that captures both those processes besides application 🤔